### PR TITLE
remove freebasics from mc2

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@calston @codiebeulaine ready for review

File: .travis.yml [hunk 1]
```diff
@@ -17,7 +17,6 @@ install:
   - pip install -r requirements.txt --cache-dir $HOME/.pip-cache/
   - pip install -e . --cache-dir $HOME/.pip-cache/
 script:
-  - flake8
```
Comment (jerith) [inline .travis.yml]: Why remove `flake8`?

Comment (jerith) [inline .travis.yml]: Oh, because pytest is doing it now.

File: requirements-dev.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
-pytest
+pytest>=2.8
```
Comment (jerith) [inline requirements-dev.txt]: What's the minimum version requirement for?

**Comment by jerith** [comment]
Do we still need the base controller now that the only thing using it is `DockerController`?

File: requirements-dev.txt (hunk 2 above)
Comment (miltontony) [inline requirements-dev.txt]: `pytest-flake8` has a minimum dependency and travis was failing because it had an older version.

Comment (jerith) [inline requirements-dev.txt]: I see the version conflict exception in the Travis build, so I guess it's the minimum version supported by `pytest-flake8`. I wonder why that doesn't specify the version itself.

**Comment by jerith** [comment]
:+1: from me.

File: requirements-dev.txt (hunk 2 above)
Comment (miltontony) [inline requirements-dev.txt]: yeah, not sure why.

**Pull request merged**